content/docs/command-reference/import-url.md

```diff
@@ -1,8 +1,9 @@
 # import-url
 
 Download a file or directory from a supported URL (for example `s3://`,
-`ssh://`, and other protocols) into the <abbr>workspace</abbr>, and track it (an
-import `.dvc` file is created).
+`ssh://`, and other protocols) into the <abbr>workspace</abbr> (or to the
```

Comment:
I think we need to work on this a bit more ... probably get rid of Download? 

brackets look too complicated for the very beginning of the doc

cc @jorgeorpinel

Reply:
Agree. "Download to remote storage" doesn't seem clear... 🤔 

> The parenthesis here look OK to me though. and that's already in https://dvc.org/doc/command-reference/import-url (out of scope to change here?)

Reply:
But OK, if we completely rewrite this anyway, we can address the parenthesis situation along the way. Maybe something like

``Track a file or directory found in an external location (`s3://`, `/local/path`, etc.), and download it to the local project, or make a copy in [remote storage](/doc/command-reference/remote).``

Reply:
Please notice similar suggestion (for add) in https://github.com/iterative/dvc/pull/5198#pullrequestreview-566897428

Reply:
p.s. but I guess let's decide here first, where the description can be a little longer, and then we can update similarly in core?